src/routes/Login/components/LoginPage/LoginPage.js

```diff
@@ -29,7 +29,8 @@ function LoginPage() {
         .doc(`${USERS_COLLECTION}/${authState.user.uid}`)
         .get();
       // Redirect to search page if user exists
-      if (userSnap.exists) {
+      if (userSnap.get('preciseLocation')) {
+        // if (userSnap.exists) {
```

Comment:
We can remove this old line if we are doing this by if preciseLocation is set yet or not